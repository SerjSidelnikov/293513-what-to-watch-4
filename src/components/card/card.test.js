import React from 'react';
import renderer from 'react-test-renderer';

import Card from './card';

const Element = () => <div>children</div>;

describe(`Card`, () => {
  it(`Card rendered correctly`, () => {
    const tree = renderer.create(
        <Card
          name={``}
          onMouseLeave={() => {}}
          onMouseEnter={() => {}}
        >
          <Element/>
        </Card>,
        {
          createNodeMock: () => {
            return {};
          }
        }
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
