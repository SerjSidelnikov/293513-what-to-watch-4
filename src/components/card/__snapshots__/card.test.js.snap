// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Card rendered correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video
      autoPlay={false}
      height="100%"
      loop={true}
      preload="none"
      width="100%"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="#"
    >
      Fantastic Beasts: The Crimes of Grindelwald
    </a>
  </h3>
</article>
`;
